main.js

```diff
@@ -8,42 +7,41 @@ import enLocaleData from "react-intl/locale-data/en";
 import FastClick from "fastclick";
 import { Provider } from "react-redux";
 import "bootstrap";
+import cockpit from "cockpit"; // eslint-disable-line global-require, import/no-unresolved
```

Comment:
For a follow-up cleanup: To avoid the eslint override, you can put a `module.exports.externals = { "cockpit": "cockpit" }`

https://github.com/cockpit-project/cockpit/blob/master/webpack.config.js#L249

But actually our webpack config already has that, so it's strange that eslint complains about that. starter-kit doesn't use these hacks.